eng/Versions.props

```diff
@@ -59,7 +59,7 @@
     <!-- Packages from dotnet/runtime -->
     <MicrosoftBclAsyncInterfacesPackageVersion>1.0.0</MicrosoftBclAsyncInterfacesPackageVersion>
     <MicrosoftWin32RegistryPackageVersion>5.0.0-preview.3.20180.15</MicrosoftWin32RegistryPackageVersion>
-    <MicrosoftExtensionsPrimitivesPackageVersion>5.0.0-preview.3-runtime.20181.3</MicrosoftExtensionsPrimitivesPackageVersion>
+    <MicrosoftExtensionsRuntimePackageVersion>5.0.0-preview.3-runtime.20181.3</MicrosoftExtensionsRuntimePackageVersion>
```

Comment:
Hmm actually does this work with darc version updates? We want to make sure these are updated automatically so I'm not sure we can just use one version property.

Reply:
I believe you should keep all the properties named the same then change which repo they come from in Version.Details.xml

Reply:
Sorry, realize this is extensions.  All properties need to be added here as PackageId.Replace('.', '') + "PackageVersion"